.github/workflows/release.yml

```diff
@@ -72,18 +72,18 @@ jobs:
         with:
           go-version: 1.18.x
       - name: Login to Docker Hub
-        uses: docker/login-action@v2
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a #v2
```

Comment:
@piksel I would also trust `docker` tbh. But this is ok too. At least it's not that madness from before.

Reply:
Yeah, both goreleaser and docker should be trusted as far as I am concerned, but since they are in the security hotspot (getting secrets and publishing) I felt that it was better to not include them in the "madness purge" 😁